chart/kubeapps/templates/_helpers.tpl

```diff
@@ -98,10 +98,10 @@ Create name for the apprepository-controller based on the fullname
 {{- end -}}
 
 {{/*
-Create name for the apprepository bootstrap job
```

Comment:
I'm assuming this is an old job which no longer exists, as I couldn't see the name being used anywhere.